packages/terra-table/src/subcomponents/_CheckMarkCell.jsx

```diff
@@ -134,28 +135,33 @@ const CheckMarkCell = ({
     { 'hide-cell': isHidden },
     { 'is-interactable': !isDisabled && isSelectable },
     { 'is-top-align': attrPadding },
+    { 'is-focused': isFocused && !isDisabled },
   );
 
   return (
     <div
       {...customProps}
       {...attrSpread}
+      tabIndex="-1"
```

Comment:
Is this index unnecessary or are we missing the equivalent in the HeaderCheckMarkCell?

Reply:
removed it here https://github.com/cerner/terra-core/pull/3126/commits/bc141da851635bf0c558673882deb0dfafdac8ca.